src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+promise1
+  .then((message) => {
+    const div = document.createElement('div');
+
+    div.classList.add('message');
+    div.textContent = message;
+    document.body.appendChild(div);
+  })
+  .catch((error) => {
+    const div = document.createElement('div');
+
+    div.classList.add('message', 'error-message');
+    div.textContent = error;
```

Comment:
When displaying the error message, consider using `error.message` instead of `error` to ensure that only the error message string is displayed, rather than the entire Error object.